libsecret/libsecret.json

```diff
@@ -2,6 +2,7 @@
   "name": "libsecret",
   "buildsystem": "meson",
   "config-opts": [
+    "-Dcrypto=gnutls",
```

Comment:
This needs gnutls >= 3.8.2, the runtime ships 3.8.0 it silently falls back to libgcrypt.

Reply:
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/merge_requests/17826

Reply:
Please propose this change in a separate MR.

Reply:
> Please propose this change in a separate PR.

Done